helix-term/src/ui/markdown.rs

```diff
@@ -37,14 +37,23 @@ impl Markdown {
             config_loader,
             text_style: "markup.normal".into(),
             block_style: "markup.raw.inline".into(),
-            heading_style: "markup.heading".into(),
+            heading_styles: [
+                "markup.heading.1".into(),
+                "markup.heading.2".into(),
+                "markup.heading.3".into(),
+                "markup.heading.4".into(),
+                "markup.heading.5".into(),
+                "markup.heading.6".into(),
+            ],
         }
     }
 
     pub fn style_group(mut self, suffix: &str) -> Self {
         self.text_style = format!("markup.normal.{}", suffix);
         self.block_style = format!("markup.raw.inline.{}", suffix);
-        self.heading_style = format!("markup.heading.{}", suffix);
+        for i in 0..self.heading_styles.len() {
+            self.heading_styles[i] = format!("markup.heading.{}.{}", i + 1, suffix);
+        }
         self
     }
```

Comment:
Not quite sure what this function is used for. I simply made it work with my changes.